src/Apps/Order/Components/__tests__/AddressVerificationFlow.jest.tsx

```diff
@@ -21,14 +23,38 @@ const componentProps = {
   onChosenAddress: mockOnChosenAddress,
 }
 
-jest.unmock("react-relay")
+jest.mock("System/useSystemContext")
 
+jest.unmock("react-relay")
 jest.mock("react-tracking")
 
+const trackEvent = jest.fn()
+
+beforeAll(() => {
+  ;(useTracking as jest.Mock).mockImplementation(() => {
+    return {
+      trackEvent,
+    }
+  })
+})
+
 const { renderWithRelay } = setupTestWrapperTL<
-  AddressVerificationFlow_Test_Query
+  AddressVerificationFlow_Test_Query & {
+    onChosenAddress: () => void
+    onClose: () => void
+  }
 >({
-  Component: AddressVerificationFlowFragmentContainer,
+  Component: props => {
+    return (
+      <AnalyticsContext.Provider
+        value={{
+          contextPageOwnerSlug: "example-order-id",
+        }}
+      >
+        <AddressVerificationFlowFragmentContainer {...(props as any)} />
+      </AnalyticsContext.Provider>
```

Comment:
Can we use something like [this](https://github.com/artsy/force/blob/e1c9df1422973db7d5e913c10c58080b2efa18fa/src/Apps/Consign/Routes/MarketingLanding/Components/__tests__/HowItWorksSteps.jest.tsx#L8-L12)?

Reply:
Yep

Reply:
I like this way to do it, is it just preference or a newer way to mock/stub out analytics context?